### ISSUE TITLE
Navigation dots not working when using scrollBar:true without anchors

### ISSUE THREAD

**Issue by peterph-am** [alvarotrigo/fullPage.js#974]
Hi,

The navigation dots in the plugin don't seem to be receiving the class active when scrolling through the slides sections.

When I load/refresh the page the first dot has the "active" class but once scrolling starts it is removed and the other links don't get any classes.

These are my settings:

````
menu: true,
navigation: true,
navigationPosition: 'left',
slidesNavigation: true,
autoScrolling: true,
continuousVertical: false,
css3: true,
easing: 'easeOutQuart',
easingcss3: 'ease',
loopBottom: false,
loopHorizontal: true,
loopTop: false,
normalScrollElementTouchThreshold: 5,
resize : false,
scrollBar: true,
scrollingSpeed: 1000,
scrollOverflow: false,
sectionSelector: _el.slides,
touchSensitivity: 5,

//events
onLeave: function(index, nextIndex, direction){
	if(index === 1 && nextIndex === 2){
		_el.header.removeClass('is_alt');
	}

	else if(index === 2 && nextIndex === 1){
		_el.header.addClass('is_alt');
	}
},
afterLoad: function(anchorLink, index){
	if(index !== 1) {
		_el.header.removeClass('is_alt');
	}
},
````

**Comment by alvarotrigo** [comment]
Yeah right, there's a bug there. 
I'm updating the title of your issue to a better one.

**Comment by peterph-am** [comment]
I see thanks for the heads up, I took removed the menu option

**Comment by alvarotrigo** [comment]
Should be fixed now in version 2.5.7.
